### ISSUE TITLE
Android Teams app cannot navigate to application tab deep link

### ISSUE THREAD

**Issue by nbelyh** [MicrosoftDocs/msteams-docs#5278]
I am building a link to my tab using deep links:
https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/build-and-test/deep-links#generate-a-deep-link-to-your-tab

My link looks like this:
`https://teams.microsoft.com/l/entity/${appId}/${entityId}?context=${encodedContext}`

I have a fully-functional, minimal application to reproduce the issue here:
https://github.com/nbelyh/teams-android-entity-id

There are manifest the file, and two app files (config.html + content.html)

The code of the tab goes like this:
```js
microsoftTeams.initialize();
microsoftTeams.getContext(teamsContext => {
  document.getElementById("clickme").addEventListener("click", () => {

    var encodedContext = encodeURI(`{"channelId": "${teamsContext.channelId}"}`);
    var appId = '<my app id here>';
    var entityId = teamsContext.entityId;
    var url = `https://teams.microsoft.com/l/entity/${appId}/${entityId}?context=${encodedContext}`;

    microsoftTeams.executeDeepLink(url);
  });
});
microsoftTeams.appInitialization.notifySuccess();
```

Basically, it is a minimalistic teams tab application that builds a deep link to navigate to itself.
And when the user clicks the button, executes the link.

The link works (when executed, application "opens" the same tab) in desktop client, Web, iOS, but **does not work on android**.
Instead of navigating to the tab, it navigates to the channel.

**Comment by Nivedipa-MSFT** [comment]
@nbelyh - We are investigating your query. We will get back to you once we have any update.

**Comment by astawkieliszek** [comment]
Hi, I suppose this is exactly the same behavior I described in issue #4028.

**Comment by nbelyh** [comment]
Yes, correct - overlookd this one 👍

**Comment by Nivedipa-MSFT** [comment]
@nbelyh - There is already bug raised for this issue. We will let you know once we get updates on it.

**Comment by sayali-MSFT** [comment]
Hello @nbelyh , The issue has been resolved. When you have a moment, could you confirm that it's fixed on your side as well?
